### PR TITLE
download_changesets_uid.sh

### PR DESCRIPTION
**Pull request by SomeoneElseOSM**
Apply similar changes as just checked in to download_changesets.sh 
Also moved start date in download_changesets.sh back to 2013 (I changed that for testing and forgot to change it back).
As before, this is just a "get it working 99% of the time again" fix - likely the way forward is https://github.com/woodpeck/osm-revert-scripts/pull/31 .

### CODE REVIEW

**Comment by AntonKhorev** [comment]
try
```
user_changesets.pl download changes --uid <uid>
```

File: Undo.pm [hunk 1]
```diff
@@ -131,7 +131,7 @@ sub determine_undo_action
     my $restore_version;
     my $override_version;
     my $override = 0;
-    my $force = 0; # if this is set to 1, any object touched by the undo userwill be reverted even if there are later modifications by others
+    my $force = 1; # if this is set to 1, any object touched by the undo userwill be reverted even if there are later modifications by others
```
Comment (AntonKhorev) [inline Undo.pm]: Wasn't this pull request about shell scripts? They don't call perl scripts.

Comment (SomeoneElseOSM) [reply Undo.pm]: It was - that change to Undo.pm shouldn't have been in here.

**Comment by SomeoneElseOSM** [comment]
No need for this any more - "user_changesets.pl" does that now.

**Pull request closed without merging**